tests/test_z_cmdline.py

```diff
@@ -9,8 +9,8 @@
 
 pytest_plugins = "pytester"
 
-from tox.session import Session
-from tox.config import parseconfig
+from tox.session import Session  # noqa #E402 module level import not at top of file
+from tox.config import parseconfig  # noqa #E402 module level import not at top of file
```

Comment:
I could, but again tried to not change its meaning 👍  should we fix this as part of this PR, or as a subsequent one?